hashtable.csv

```diff
@@ -15,6 +15,7 @@ keccak-256,         ,                         0x1B
 keccak-384,         ,                         0x1C
 keccak-512,         ,                         0x1D
 murmur3,            ,                         0x22
+murmur3-32          ,                         0x23
```

Comment:
Could you update https://github.com/multiformats/multicodec/blob/master/table.csv too?